services/github/src/handlers/index.js

```diff
@@ -128,12 +125,8 @@ class Handlers {
       githubEvents.taskGroupCreationRequested(`route.${this.context.cfg.app.statusTaskRoute}`),
     ];
 
-    // Listen for taskDefined event to create initial status on github
-    const taskBindings = [
-      queueEvents.taskDefined(`route.${this.context.cfg.app.checkTaskRoute}`),
-    ];
-
-    const callHandler = (name, handler) => message => {
+    // handler returned must be async, as timedHandler will not be able to time it correctly
+    const callHandler = (name, handler) => message =>
```

Comment:
🦅 👁️

Reply:
this caused function to return early, and all `monitor.timedHandler()` were returning over-optimistic durations :)